src/Stripe.net/Services/Checkout/Sessions/SessionCreateOptions.cs

```diff
@@ -30,20 +28,20 @@ public class SessionCreateOptions : BaseOptions, IHasMetadata
         [JsonProperty("client_reference_id")]
         public string ClientReferenceId { get; set; }
 
-        /// <summary>
-        /// The email address used to create the customer object. If you already know your
-        /// customer’s email address, use this attribute to prefill it on Checkout.
-        /// </summary>
-        [JsonProperty("customer_email")]
-        public string CustomerEmail { get; set; }
-
         /// <summary>
         /// ID of the customer this Checkout Session is for if one exists. May only be used with
         /// <c>LineItems</c>. Usage with <c>SubscriptionData</c> is not yet available.
         /// </summary>
         [JsonProperty("customer")]
         public string Customer { get; set; }
 
+        /// <summary>
+        /// The email address used to create the customer object. If you already know your
+        /// customer’s email address, use this attribute to prefill it on Checkout.
+        /// </summary>
+        [JsonProperty("customer_email")]
+        public string CustomerEmail { get; set; }
```

Comment:
cc @ob-stripe I never understand whether `customer_email` / `CustomerEmail` comes before/after `customer`/`Customer`
Fairly sure that in Go I chose to do the opposite.

Reply:
This is the sort order using `Array#sort` in javascript:

```js
['customer_email', 'customer', 'customer_phone'].sort((a, b) => a.localeCompare(b))
// => ["customer", "customer_email", "customer_phone"]
```

Which I think is what we want as that's what'll be used when ordering with autogen.

Reply:
Yeah, this is just how strings are sorted. If it helps, you can think of the null terminator byte as part of the string for sorting purposes. `customer\x00` should appear before `customer_email\x00`.